src/smc-webapp/course/actions.ts

```diff
@@ -1492,42 +1497,104 @@ export class CourseActions extends Actions<CourseState> {
     });
   }
 
-  set_grade(assignment, student, grade) {
-    let left;
+  clear_edited_feedback = (
+    assignment: AssignmentRecord,
+    student: StudentRecord
+  ) => {
     const store = this.get_store();
-    if (store == null) {
+    if (store == undefined) {
       return;
     }
-    assignment = store.get_assignment(assignment);
-    student = store.get_student(student);
-    const obj = {
-      table: "assignments",
-      assignment_id: assignment.get("assignment_id")
-    };
-    const grades = (left = this._get_one(obj).grades) != null ? left : {};
-    grades[student.get("student_id")] = grade;
-    (obj as any).grades = grades;
-    this._set(obj);
-  }
 
-  set_comments(assignment, student, comments) {
-    let left;
+    const open_grades = store.get("active_feedback_edits");
+    const new_open_grades = open_grades.delete(
+      assignment.get("assignment_id") + student.get("student_id")
```

Comment:
I'm curious. Should this line maybe call `utils::assignment_identifier` to semantically express that this is the common key used everywhere?

Reply:
Oh, yup. Missed that one.